yarn.lock

```diff
@@ -18646,7 +18635,7 @@ merkle-patricia-tree@^3.0.0:
     rlp "^2.0.0"
     semaphore ">=1.0.1"
 
-mersenne-twister@^1.0.1:
+mersenne-twister@^1.0.1, mersenne-twister@^1.1.0:
```

Comment:
Would be nice if we could get an updated version of `polyfill-crypto.getrandomvalues` that used `mersenne-twister@^1.1.0` so we wouldn't duplicate this dep.

Reply:
These are both being resolved to the same version, so it is currently being deduplicated. The range `polyfill-crypto.getrandomvalues` asks for is `^1.0.0`, so `1.1.0` is a compatible version.